rn-cli.config.js

```diff
@@ -7,6 +8,10 @@ enm.canvas = __dirname + '/__mocks__/nodejsMock.js';
 
 module.exports = {
 	extraNodeModules: enm,
+	getBlacklistRE: function() {
+		// blacklist the GB packages we want to consume from NPM (online) directly
+		return blacklist( [ /gutenberg\/packages\/(autop|hooks|i18n|is-shallow-equal)\/.*/ ] );
+	},
```

Comment:
Again here - I got more context from a chat elsewhere about this being blacklisted as we think we won't need to modify source files for. That makes a lot of sense, but just by reading code alone (not even this PR, but the code in a general way) it's difficult to understand that we are using the source files and blacklisting a subset of modules for the purpose of ease at modifying while developing. Can we add a nice comment in code that connects this part and <_wherever it's indicated in code we're using source files_>?

Reply:
Fair point. The packages we currently load from source have been chosen a bit arbitrarily, except for the `element` package in which we already have native code and so it's more likely we'll want to modify-and-try out fast. The other packages we use from source (`data` and `deprecated`) don't have native code and so, I just went ahead and made them resolved from NPM (with 8e5ebc8).

> Can we add a nice comment in code that connects this part and <wherever it's indicated in code we're using source files>?

I added a relevant comment (with 08f3626) but notice that effectively, there's nowhere something indicating that a package is loaded directly from source. What happens is that Metro automagically figures out that a module imported in code ([example](https://github.com/wordpress-mobile/gutenberg-mobile/blob/master/src/globals.js#L3)) can be satisfied by the package found inside the `gutenberg/packages/<module>` folder.